cdc/sink/dmlsink/mq/dispatcher/topic/expression.go

```diff
@@ -94,3 +102,30 @@ func (e Expression) Substitute(schema, table string) string {
 		return topicName
 	}
 }
+
+// PulsarValidate checks whether a pulsar topic name is valid or not.
+func (e Expression) PulsarValidate() error {
+	// validate the topic expression
+	topicName := string(e)
+
+	if len(topicName) == 0 {
+		return errors.ErrPulsarInvalidTopicExpression.GenWithStackByArgs(
+			"topic name is empty")
+	}
+
+	// if not full name, must be simple name
+	if !pulsarTopicNameREFull.MatchString(topicName) {
```

Comment:
Prefer to use `require` to check if a case meet expectation, like:https://github.com/pingcap/tiflow/blob/9a5bc3b5b0c99dfaccf36c9dcae8c6471bdf5c0f/cdc/sink/dmlsink/mq/dispatcher/topic/expression_test.go#L218 . I think it's more clear.

Reply:
ok，I will